src/components/cardbuilder/card.css

```diff
@@ -503,14 +503,14 @@ button::-moz-focus-inner {
 
     .squareCard,
     .portraitCard {
-        width: 33.333333333333333333333333333333%;
+        width: 50%;
```

Comment:
We definitely can't make changes to the main card builder style, so unless you have a different plan for this, we should probably wait until we convert more of these items to custom cards.

Reply:
Yeah, I think I'll close it for now because I don't have a plan atm, and maybe come back to it in the future